v22.1/create-security-certificates-custom-ca.md

```diff
@@ -164,8 +159,12 @@ To enable certificate revocation:
       In the `strict` mode, all certificates are presumed to be invalid if the OCSP server is not reachable. Setting the cluster setting `security.ocsp.mode` to `strict` will lock you out of your CockroachDB database if your OCSP server goes down.
       {{site.data.alerts.end}}
 
+
 ## See also
 
+- [Public Key Infrastructure (PKI) and Transport Layer Security (TLS)](security-reference/transport-layer-security.html)
```

Comment:
This is a lot of links -- I wonder if you could prune one or two. https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#see-also